.github/workflows/codecov.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         run: flutter test --coverage
       - name: Upload coverage file
         run: |
-            curl -Os https://uploader.codecov.io/latest/linux/codecov
```

Comment:
Can you please remove these spaces to prevent making change?

Reply:
Done thanks to get this error